### DISCUSSION TITLE
ELN Consortium meeting 8

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [TheELNConsortium/TheELNFileFormat#90] [General]
Thursday January 9th 2025 at 3PM CET.

https://zoom.us/j/94151960099?pwd=Ehac6ixJytQaAvePJ8yxpljkLjozXa.1

**Comment by salexan2001** [comment]
Unfortunately I cannot make it today.

**Comment by NicolasCARPi** [comment]
quick notes of the meeting:
      
# Discussion about hierarchical data structure for nested Datasets

https://github.com/TheELNConsortium/TheELNFileFormat/issues/98
how to handle nested Datasets?
we should go with flat system.
update spec: all Datasets that are meant to be imported must be children (in the hasPart) of the root node ./
and when looking at hasPart of Dataset, ignore the ones that are @type Datasets
see PR: https://github.com/TheELNConsortium/TheELNFileFormat/pull/113

# Misc

* for history of changes: how to handle older versions?
* to bind a File to variablemeasured, which can only be on a Dataset, use the about property of File
File is the associatedMedia of a Dataset that is part of a main Dataset
* documentation: will use gh pages
* created PR on RO-Crate repository/doc to add ELN as a packaging solution for RO-Crate: https://github.com/ResearchObject/ro-crate/pull/389

# Schedule
Meeting every last tuesday every two months

Next: 25th march 3PM CET
